### PR TITLE
Feature/friendly readme

### PR DESCRIPTION
**Pull request by cshcomcom**
## Description
- Update README!

### CODE REVIEW

File: src/ain.ts [hunk 1]
```diff
@@ -139,6 +139,10 @@ export default class Ain {
     return result;
   }
 
+  /**
+   * Sends signed transactions to the network
```
Comment (platfowner) [inline src/ain.ts:143]: minor: period(.)?

File: src/ain-db/ref.ts [hunk 2]
```diff
@@ -355,7 +359,7 @@ export default class Reference {
   // }
 
   /**
-   * Returns a get request
+   * Returns a get request.
```
Comment (platfowner) [inline src/ain-db/ref.ts:362]: minor: s/Returns/Builds/?

File: src/ain-db/ref.ts [hunk 3]
```diff
@@ -41,6 +41,10 @@ export default class Reference {
     this._isGlobal = false;
   }
 
+  /**
+   * Sets a global mode.
```
Comment (platfowner) [inline src/ain-db/ref.ts:45]: minor: 'Sets _isGlobal value'?

Comment (platfowner) [reply src/ain-db/ref.ts:45]: or 'Set the global path flag'?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,276 @@
...
+
+### Ain.Database
+
+`ref(path?: string): Reference`
+- Returns a reference instance of the given path.
+
+#### Ain.Database.Reference
+
+`setIsGlobal(isGlobal: boolean)`
+- Sets a global mode.
```
Comment (platfowner) [inline README.md:104]: minor: 'Sets the global path flag.'?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,276 @@
...
+- Returns the function configs that are related to the input ref.
+
+`matchRule(params?: MatchInput): Promise<any>`
+- Returns the rule configs that are related to the input ref.
+
+`matchOwner(params?: MatchInput): Promise<any>`
+- Returns the owner configs that are related to the input ref.
+
+`static buildGetRequest(type: GetOperationType, ref: string)`
+- Returns a get request.
```
Comment (platfowner) [inline README.md:167]: minor: 'Builds a get request.'?

**Review by platfowner — APPROVED** [review]
LGTM with some minor coments.
Thanks!

**Comment by cshcomcom** [comment]
Thanks for the review! All comments are addressed! Now merging!

**Pull request merged**